package.json

```diff
@@ -13,7 +13,7 @@
     "postinstall": "lerna bootstrap --stream",
     "lint": "yarn lint:css && yarn lint:js && yarn lint:markdown",
     "lint:css": "stylelint packages/*/src/**/*.js && stylelint utils/**/*.js",
-    "lint:js": "eslint \"packages/*/src/\" \"utils/\" --max-warnings 0",
+    "lint:js": "eslint \"packages/*/src/**/*.js\" \"utils/**/*.js\" --max-warnings 0",
```

Comment:
This has the effect of skipping JS files directly under package `src/` directories. Use the following syntax for this script:

```json
"lint:js": "eslint packages/*/src/ utils/ --max-warnings 0"
```

(dispense with double-quotes, etc.)

Reply:
Updated.